sktime/registry/_tags.py

```diff
@@ -124,6 +124,42 @@
         ("list", ["pd.Series", "pd.DataFrame", "np.array"]),
         "which machine type(s) is the internal _fit/_predict able to deal with?",
     ),
+    # "capability:multivariate" is the inverse of "univariate-only" tag.
+    # They are kept distinct intentionally for easier TSC refactoring.
+    # Will be merged after refactor completion.
+    (
+        "capability:multivariate",
+        "classifier",
+        "bool",
+        "can classifier use exogeneous data (X)?",
```

Comment:
that's not exactly true - it should mean "can classifier classify time series with 2 or more variables?"

Reply:
bdd99a4a